### PR TITLE
feat: Add basic fine tuning page UI

### PR DESCRIPTION
**Pull request by lynnagara**
Preparing the UI elements for the fine tuning pages

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by Chrissy** [comment]
🎉 🎉 🎉 🎉

File: src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            name: project.slug,
+            label: project.name,
+          };
+        }),
+      };
+    });
+  }
+
+  renderBody() {
+    if (this.state.loading) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx]: I think `AsyncView` handles this

File: src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    const {path} = this.props.route;
+    const data = this.getFieldData(path, this.state.projects);
+
+    return (
+      <div>
+        <ApiForm apiMethod="PUT" apiEndpoint={'/users/me/notifications/'}>
+          <Panel>
+            <PanelHeader>
+              <div className="text-light">{this.props.route.name}</div>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx]: Can we use a styled component here?

File: src/sentry/static/sentry/app/routes.jsx [hunk 3]
```diff
@@ -116,14 +117,21 @@ function appendTrailingSlash(nextState, replaceState) {
   }
 }
 
-const accountSettingsRoutes = [
-  <IndexRedirect key="account-settings-index" to="notifications/" />,
+const accountSettingsDetailedRoutes = [
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: nit: I would just nest this inline inside of `notifications/` route. The other route arrays were so we could re-use route trees, which we aren't doing with these.

**Review by billyvg — APPROVED** [review]
Looks good, let's add an acceptance test for this. Also it would help when reviewing PRs to attach a screenshot.

File: src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx]: Yep. I'm gonna merge as-is for now though, and ask @Chrissy for help on the final styling of these components

**Pull request merged**